sys/linux/bpf.txt

```diff
@@ -22,7 +22,7 @@ type btf_type_id int32[1:5]
 type btf_opt_type_id int32[0:5]
 
 # NEED: offset in bpf_btf_program:strings. We can't express this, so we just use a small index.
-type btf_name_off int32[1:16]
+type btf_name_off int32[0:16]
```

Comment:
We have `btf_opt_name_off` for this (see 1 line below :)).
My impression was that string 0 is reserved as "empty" and in some context we need a non-empty string (index>0). Thus we have btf_name_off and btf_opt_name_off for cases where empty string is acceptable.

Reply:
Ah, right. If only I had searched how the next line was used... Sorry for the noise!